website/docs/automation/gitops/argocd/deploy.md

```diff
@@ -24,16 +24,24 @@ $ kubectl create ns argocd-demo
 Create an Argo CD application:
 
 ```bash
-$ argocd app create argocd-demo --repo https://github.com/aws-samples/eks-workshop-v2.git --path environment/workspace/manifests/catalog --dest-server https://kubernetes.default.svc --dest-namespace argocd-demo
+$ argocd app create argocd-demo --repo https://github.com/aws-samples/eks-workshop-v2.git --path environment/workspace/modules/automation/gitops/argocd --dest-server https://kubernetes.default.svc --dest-namespace argocd-demo
 $ application 'argocd-demo' created
```

Comment:
You should remove the $ on the second line so that it is not copy

Reply:
good catch! I removed the $ sign